src/test/java/lotto/domain/LottoStatisticsTest.java

```diff
@@ -22,17 +22,17 @@ void init() {
     @Test
     void calculateMatchingCounts() throws Exception {
         //given
-        List<LottoNumber> lottoNumberList = new ArrayList<>();
-        lottoNumberList.add(new LottoNumber(Arrays.asList(1, 2, 3, 4, 5, 6)));
-        lottoNumberList.add(new LottoNumber(Arrays.asList(1, 2, 3, 10, 11, 12)));
-        lottoNumberList.add(new LottoNumber(Arrays.asList(1, 2, 3, 7, 8, 9)));
-        LottoNumbers lottoNumbers
-                = new LottoNumbers(lottoNumberList, new RandomNumberCreation());
-        WinningNumber winningNumber = new WinningNumber(Arrays.asList(1, 2, 3, 4, 5, 6));
+        List<LottoTicket> lottoTicketList = new ArrayList<>();
```

Comment:
변수 이름에 `-List`와 같은 자료형을 붙이면, 추후 요구사항이 변경되어 자료형이 변경될 경우
변수의 이름도 함께 변경해야 하는 번거로움이 있습니다.
자료형이 컬렉션이라면 이름을 복수형으로 짓는 것을 추천드리고 싶습니다.
```suggestion
        List<LottoTicket> lottoTickets = new ArrayList<>();
```